config/locales/fr.yml

```diff
@@ -39,6 +39,30 @@ fr:
     posts: Statuts
     remote_follow: Suivre à distance
     unfollow: Ne plus suivre
+  admin:
+    settings:
+      click_to_edit: Cliquez pour éditer
+      contact_information:
+        email: Entrez une adresse courriel publique
```

Comment:
Ah, might be, I didn't see that. I agree we need consistency... I just think email is more common. But if everything else says courriel let's keep that I guess.